### DISCUSSION TITLE
Hacktoberfest project

### DISCUSSION THREAD

**Discussion by SaptarshiSarkar12** [SaptarshiSarkar12/Drifty#2] [Announcements]
This project has been labelled as hacktoberfest project. So, anyone participating in hacktoberfest can contribute here. This project also welcomes contributions from those who are not participating. Just make an issue if its not there and then make a PR linking the issue.

**Comment by SaptarshiSarkar12** [comment]
Please follow the contributing guidelines and the code of conduct.

**Comment by BoseSj** [comment]
wouldn't it be nice to move the `Contributing` section after the `Demo`, as done in all well-known project READMEs? As the page would refer to users before it ll to potential contributors?

**Reply by SaptarshiSarkar12** [reply]
Yeah @BoseSj ❕ Please open an issue and a PR to make the required changes. Thanks for your suggestion ❕.

**Reply by BoseSj** [reply]
Okay😄 I am on it.

**Reply by SaptarshiSarkar12** [reply]
Great @BoseSj ❕

**Reply by BoseSj** [reply]
Done😄

**Reply by BoseSj** [reply]
I am thinking about making some changes in README

_Currently_
![image](https://user-images.githubusercontent.com/58129377/194591451-d8dd3bff-92aa-41e0-b526-c5479921ef2e.png)

I want to make some changes to make it look like this >>
![image](https://user-images.githubusercontent.com/58129377/194591523-c8564861-7384-4730-9614-e6008b0abb28.png)

It looks better to me. 

What do u @SaptarshiSarkar12  think?

**Reply by BoseSj** [reply]
I will be removing the `issue` and `discussion` badges from there, as one can see those already in the GitHub interface easily. Plus, it's cluttering the space.

Let me know what u think.

**Reply by SaptarshiSarkar12** [reply]
Yes @BoseSj . Its a great idea!
